CoffeeChat/Controller/EditLocationAvailabilityViewController.swift

```diff
@@ -0,0 +1,296 @@
+
+        scheduleLocationSubLabel.text = "pick three"
+        scheduleLocationSubLabel.font = ._16CircularStdBook
+        scheduleLocationSubLabel.textColor = .greenGray
+        view.addSubview(scheduleLocationSubLabel)
+
+        let locationsCollectionViewLayout = UICollectionViewFlowLayout()
+        locationsCollectionViewLayout.minimumLineSpacing = locationLineSpacing
+        locationsCollectionViewLayout.minimumInteritemSpacing = locationinteritemSpacing
+
+        locationsCollectionView = UICollectionView(frame: .zero, collectionViewLayout: locationsCollectionViewLayout)
+        locationsCollectionView.allowsMultipleSelection = true
+        locationsCollectionView.showsVerticalScrollIndicator = false
+        locationsCollectionView.showsHorizontalScrollIndicator = false
+        locationsCollectionView.delegate = self
+        locationsCollectionView.dataSource = self
+        locationsCollectionView.backgroundColor = .clear
+        locationsCollectionView.layer.masksToBounds = false
+        locationsCollectionView.register(SchedulingPlaceCollectionViewCell.self, forCellWithReuseIdentifier: campusReuseId)
+        locationsCollectionView.register(SchedulingPlaceCollectionViewCell.self, forCellWithReuseIdentifier: ctownReuseId)
+        locationsCollectionView.register(LocationHeaderLabel.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: campusHeaderId)
+        locationsCollectionView.register(LocationHeaderLabel.self, forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader, withReuseIdentifier: ctownHeaderId)
+        locationsCollectionView.isScrollEnabled = false
+        view.addSubview(locationsCollectionView)
+
+        selectedCampusLocations = campusLocations.filter {savedLocations.contains($0)}
+        selectedCtownLocations = ctownLocations.filter {savedLocations.contains($0)}
+
+        locationSections = [
+            .campus(campusLocations),
+            .ctown(ctownLocations)
+        ]
+
+        setupConstraints()
+    }
+
+    private func setupNavigationBar() {
+        navigationController?.navigationBar.barTintColor = .backgroundLightGreen
+        navigationController?.navigationBar.titleTextAttributes = [
+            .font: UIFont.getFont(.medium, size: 24)
+        ]
+
+        backButton.setImage(UIImage(named: "backArrow"), for: .normal)
+        backButton.snp.makeConstraints { make in
+            make.size.equalTo(CGSize(width: 10, height: 20))
+        }
+        backButton.addTarget(self, action: #selector(backPressed), for: .touchUpInside)
+        navigationItem.leftBarButtonItem = UIBarButtonItem(customView: backButton)
+
+        saveBarButtonItem.title = "Save"
+        saveBarButtonItem.tintColor = .darkGreen
+        saveBarButtonItem.target = self
+        saveBarButtonItem.action = #selector(saveAvailability)
+        saveBarButtonItem.setTitleTextAttributes([
+            .font: UIFont.getFont(.medium, size: 20)
```

Comment:
Just checking, is `UIFont.getFont(.medium, size: 20)` what we want?

Reply:
yes, the Figma has 20px medium font.